_includes/patterns/race-and-ethnicity-1.md

```diff
@@ -0,0 +1,44 @@
+{% include patterns/race-and-ethnicity-intro.md variant="input" %}
+
+    {% for category in data.categories %}
+      <fieldset class="usa-fieldset margin-top-1" name="{{ category.name | slugify }}-alt">
+        <legend>
+          <div class="usa-checkbox">
+            <input class="usa-checkbox__input" type="checkbox" id="{{ category.name | slugify }}-checkbox-alt" name="{{ category.name | slugify }}-checkbox-alt" aria-describedby="details">
+            <label class="usa-checkbox__label text-bold" for="{{ category.name | slugify }}-checkbox-alt">{{ category.name }}</label>
+          </div>
+        </legend>
+        <div class="padding-left-4">
+          {% if category.types %}
+          <fieldset class="usa-fieldset margin-top-05">
+            <legend class="usa-legend font-lang-2xs text-italic" id="details">Provide details below</legend>
```

Comment:
"Provide details below" is missing a period here and in the other instances. Was that intentional? 
I am also wondering about the use of italics. I know in the example shared in OMB guidance, they used italics, but I'm not sure if that was recommended or primarily for print format.